### ISSUE TITLE
Cyclestreets only spins up one core per minute

### ISSUE THREAD

**Issue by mem48** [nptscot/npt#270]
The cyclestreets batch service appears to only allocate one core per minute to a new job up to a max of 20 cores.

For a large job, this means the routing speed is on average 50% of the max speed for the first 20 minutes. Thus 10 minutes of unnecessary waiting is added to each job. 

We currently send 8 jobs per build (4x trip plans (fastest, quietest, ebike, balanced), and 2x trip purposes (school, commute)). So that means 80 minutes of delay in each build. We are planning at least one more trip purpose so the delay will increase to 120 minutes!

@mvl22 and @si-the-pie can anything be done about this?

**Comment by mvl22** [comment]
Yes, I've been wanting to amend the thread spin-up routine for a while now. Will have another look at this soon.

**Comment by mvl22** [comment]
Just to say that the initial spin-up rate is now at 6x previous.

**Comment by Robinlovelace** [comment]
Fantastic, many thanks Martin!
